### DISCUSSION TITLE
Galaxy Brain

### DISCUSSION THREAD

**Discussion by sleepinzombie** [drknzz/GitHub-Achievements#381] [Q&A] [answered]
Hmmm...

**Comment by sleepinzombie** [comment]
Is this?

**Reply by farhansadikgalib** [reply]
yeah

**Reply by Shubhamraut01** [reply]
https://github.com/drknzz/GitHub-Achievements/discussions/381

**Reply by sleepinzombie** [reply]
@farhansadikgalib Can I have a thread to add a reply too?

**Comment by Coolbreeze151** [comment, marked as answer]
To get the Galaxy Brain achievement you need to answer at least 2 discussions and have the OP accept your answers.

If you would like, we can exchange with each other.

https://github.com/Schweinepriester/github-profile-achievements/discussions/945#discussioncomment-10782175

https://github.com/Schweinepriester/github-profile-achievements/discussions/946#discussioncomment-10782183

**Reply by sleepinzombie** [reply]
@Coolbreeze151 your above discussions are locked. I can't add comments in these anymore.
